networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,64 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../networking/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-4"]
+== Network Observability Operator 1.4.0
+The following advisory is available for the Network Observability Operator 1.4.0:
+
+* <new link here>
+
+
+[id="network-observability-operator-1.4.0-features-enhancements"]
+=== New features and enhancements
+
+[id="network-observability-without-loki-1.4"]
+==== Network Observability without Loki
+You can now install the Network Observability Operator without Loki and only export flows to KAFKA or IPFIX. For more information, see xref:../../networking/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki]. 
+
+[id="network-observability-enhanced-configuration-and-ui-1.4"]
+==== Notable enhancements
+The 1.4 release of the Network Observability Operator adds improvements and new capabilities to the {product-title} web console plugin and the Operator configuration. 
+
+This release introduces the following web console enhancements:
+
+* In the *Quick Filters*, the *Duplicate flows* filter is added to choose whether or not to show duplicated flows.
+* You can now filter source and destination traffic with image:arrow-up-long-solid.png[,10] *One-way*, image:arrow-up-long-solid.png[,10] image:arrow-down-long-solid.png[,10] *Back-and-forth*, and *Swap* filters. 
+
+For more information, see xref:../../networking/network_observability/observing-network-traffic.adoc#network-observability-quickfilter_nw-observe-network-traffic[Configuring Quick Filters].
+ 
+This release introduces the following configuration enhancements:
+
+* The `spec.processor.clusterName` parameter is added so that the name of the cluster appears in the flows data. This is useful in a multi-cluster context. When using {product-title}, leave empty to make it automatically determined.
+* You now have the option to specify different namespaces for all Network Observability components using the `spec.loki.tls.caCert.namespace` and `spec.exporters.kafka.tls.caCert.namespace` fields.
+
+For more information, see xref:../../networking/network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource] and xref:../../networking/network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference].
+
+[id="network-observability-dns-tracking-1.4"]
+==== DNS tracking
+In 1.4, you can enable DNS tracking with eBPF tracepoint hooks. With Network Observability DNS tracking, you can monitor your network, conduct security analysis, and troubleshoot DNS issues in the *Network Traffic* and *Overview* views in the web console. 
```

Comment:
I think I may have this _slightly_ backwards...
>  you can enable DNS tracking with eBPF tracepoint hooks

should be

> The Network Observability Operator makes use of eBPF tracepoint hooks to enable DNS tracking

Yes? No?

Reply:
yes, +1, so the userd aren't misguided into thinking they would have to do fancy eBPF stuff themselves :-)

Reply:
Addressed! TY!